mungegithub/mungers/submit-queue.go

```diff
@@ -650,6 +667,88 @@ func (sq *SubmitQueue) updateHealth() {
 	}
 }
 
+func (sq *SubmitQueue) getRunningPRNumber() int {
+	defer sq.Unlock()
```

Comment:
Thanks for pointing this out. I'll correct that in a follow up since it should still work until then.